permissions/plugin-aws-device-farm.yml

```diff
@@ -5,6 +5,7 @@ paths:
 - "org/jenkins-ci/plugins/aws-device-farm"
 developers:
 - "nikdabn"
-- "chabom"
-- "oleg_nenashev"
-- "patcadelina"
+- "aws_device_farm_jenkins"
+security:
+  contacts:
+    email: "aws-security@amazon.com"
```

Comment:
```suggestion
```

Is the email address associated with aws_device_farm_jenkins aws-security@amazon.com? If so, the security section is not needed at all.

Reply:
This is a known contact for other plugins maintained by Amazon AWS folks. I approve.

(Also not the same email address.)

Reply:
@slide No. The email associated with aws_device_farm_jenkins is different. 
Thank you @daniel-beck